Vic2ToHoI4/Data_Files/DataFiles/CodenamesOperatives/RME_operative_codenames.txt

```diff
@@ -0,0 +1,37 @@
+RME_OPERATIVE_CODENAME_HISTORICAL = {
```

Comment:
The folder location for this one is very confused. Also, the existing location looks to be wrong (something I did during a cleanup). The converter looks in configurables/CodenamesOperatives/ for all such files.

Reply:
Moved the file